tanner/server.py

```diff
@@ -106,6 +106,7 @@ async def on_shutdown(self, app):
         await self.session_manager.delete_sessions_on_shutdown(self.redis_client, self.pg_client)
         self.redis_client.close()
         await self.redis_client.wait_closed()
+        await self.pg_client.wait_closed()
```

Comment:
why you call only `wait_closed` without closing the pool? You closed it somewhere else?
https://aiopg.readthedocs.io/en/stable/core.html#aiopg.Pool.close

Reply:
Sorry I missed that :sweat_smile: